ggml.c

```diff
@@ -16556,14 +16539,18 @@ static thread_ret_t ggml_graph_compute_thread(void * data) {
         } else {
             // wait for other threads to finish
             const int last = node_n;
+
+            const bool do_yield = last < 0 || cgraph->nodes[last]->op == GGML_OP_MUL_MAT;
```

Comment:
Does this assume that GGML_OP_MUL_MAT is the only op that calls into blas?  If so, that isn't right: see ggml_compute_forward_out_prod_f32().

Reply:
It probably should be added, but I have a benchmark just for `GGML_OP_MUL_MAT` at the moment, so cannot verify it atm. If you observe that it improves the performance, we can add it